pkg/odo/cli/component/create.go

```diff
@@ -251,12 +251,12 @@ func (co *CreateOptions) Complete(name string, cmd *cobra.Command, args []string
 		}
 		co.CreateArgs.Name = ui.EnterComponentName(defaultComponentName)
 
-		advancedOptionsNeeded := ui.SelectYesNo("Do you wish to set advanced options")
+		advancedOptionsNeeded := ui.Proceed("Do you wish to set advanced options")
 		if advancedOptionsNeeded {
 			co.CreateArgs.Ports = ui.EnterPorts()
 			co.CreateArgs.EnvVars = ui.EnterEnvVars()
 
-			resourceLimitsNeeded := ui.SelectYesNo("Do you wish to set resource limits")
+			resourceLimitsNeeded := ui.Proceed("Do you wish to set resource limits")
 			if resourceLimitsNeeded {
```

Comment:
Nit-picking but how about: `if ui.Proceed…` then now? :)

Reply:
Not sure I follow :(

Reply:
Sorry yeah, now I get get it - I'll fix it :)

Reply:
Fixed